reorder_python_imports.py

```diff
@@ -671,22 +680,33 @@ def _version_removals(args: argparse.Namespace) -> Generator[str, None, None]:
 ]
 # END GENERATED
 
+# GENERATED VIA generate-mypy_extensions-info
+# Using mypy_extensions==0.4.3
+MYPY_EXTENSIONS_RENAMES = [
+    'mypy_extensions=typing:NoReturn',
+    'mypy_extensions=typing:TypedDict',
+]
+# END GENERATED
+
 
 def _is_py3(args: argparse.Namespace) -> bool:
-    for py, _ in FUTURE_IMPORTS:
+    for py, _ in FUTURE_IMPORTS + MYPY_EXTENSIONS_IMPORTS:
         if py.startswith('py3') and getattr(args, f'{py}_plus'):
             return True
     else:
         return False
 
 
 def _version_replaces(args: argparse.Namespace) -> List[ImportToReplace]:
-    if _is_py3(args):
-        renames = MOCK_RENAMES + SIX_RENAMES
-        return [_validate_replace_import(s) for s in renames]
-    else:
+    if not _is_py3(args):
         return []
 
+    renames = MOCK_RENAMES + SIX_RENAMES
+    if getattr(args, 'py38_plus'):
```

Comment:
this getattr does nothing, it's the same as `args.py38_plus`

Reply:
flake8-bugbear B009!